File: flutter_todo/lib/controller/dto/cmRespDto.dart
class CMRespDto {
  final int? statusCode;
  final String? msg;
  final dynamic data;

  CMRespDto({this.statusCode, this.msg, this.data});

  CMRespDto.fromJson(Map<String, dynamic> json)
      : statusCode = json["statusCode"],
        msg = json["msg"],
        data = json["data"];
}

File: todo_back/src/main/java/com/cos/todo/controller/dto/CMRespDto.java
package com.cos.todo.controller.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class CMRespDto<T> {

	private int statusCode;
	private String msg;
	private T data;
	
}
